packages/auth/lib/index.js

```diff
@@ -150,6 +155,16 @@ class FirebaseAuthModule extends FirebaseModule {
     }
   }
 
+  async setTenantId(tenantId) {
+    if (!isString(tenantId) && !isNull(tenantId)) {
+      throw new Error(
+        "firebase.auth().setTenantId(*) expected 'tenantId' to be a string or null value",
+      );
+    }
+    await this.native.setTenantId(tenantId);
```

Comment:
Hmm - the native methods do not take a Promise as an argument though, do they? They appear to be synchronous. So I'm not sure we should 'await' here if they keep their synchronous API signatures, and this method should not be async

Reply:
Here I just following same pattern with LanguageCode, and test to make this one as sync method, Then I met the issue can't find method tenantId.

Reply:
That is a fair point about following setLanguageCode - I accept that, thank you!